servers/src/main/java/tachyon/master/InodeFile.java

```diff
@@ -54,8 +62,12 @@ static InodeFile loadImage(ImageElement ele) throws IOException {
     final String ufsPath = ele.getString("ufsPath");
     final int dependencyId = ele.getInt("depId");
     final long lastModificationTimeMs = ele.getLong("lastModificationTimeMs");
+    final String username = ele.getString("username");
+    final String groupname = ele.getString("groupname");
+    final short permission = ele.getShort("permission");
 
```

Comment:
This question I remember asked before. This is a compatible problem. If the new Tachyon version runs on the older image data. There exists some issue. I want to add a version check.
if the image data is generated by older tachyon version, I will skip get the permission information from image data, currently I don't know add what kind of permission. We can talk about this question on the meeting.